__tests__/app.test.js

```diff
@@ -339,3 +339,45 @@ describe("PATCH api/reviews/:review", () => {
       });
   });
 });
+describe("GET api/reviews/(queries)", () => {
+  test("returns", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction&sort_by=review_id&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: "social deduction",
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+  test("status:400, responds with an error message when passed none valid category", () => {
+    return request(app)
+      .get("/api/reviews?category=999999")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  test("status:400, responds with an error message when passed a category that exsist but has no reviews", () => {
```

Comment:
This should return a 200 as it is a valid category but it will come back with a status 200 and an empty array